Enable initial reset for LaTeX classes to load the class mode
at creation of the document.  The declared variable is called
LaTeX-global-class-files and not TeX-global-class-files.

---
 latex.el |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- latex.el
+++ latex.el	2024-05-07 06:47:32.754527641 +0000
@@ -2706,7 +2706,7 @@ Initialized once at the first time you p
 May be reset with `\\[universal-argument] \\[TeX-normal-mode]'.")
 
 ;; Add the variable to `TeX-normal-mode-reset-list':
-(add-to-list 'TeX-normal-mode-reset-list 'TeX-global-class-files)
+(add-to-list 'TeX-normal-mode-reset-list 'LaTeX-global-class-files)
 
 (defcustom TeX-arg-input-file-search t
   "If `TeX-arg-input-file' should search for files.
